src/applications/personalization/dashboard/tests/components/debts/CopaysCard.unit.spec.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { expect } from 'chai';
+import { format } from 'date-fns';
+import { renderWithStoreAndRouter } from '~/platform/testing/unit/react-testing-library-helpers';
+
+import CopaysCard from '../../../components/debts/CopaysCard';
+
+const initialState = {
+  allDebts: {
+    isLoading: false,
+    copays: [],
+  },
+};
+
+describe('<CopaysCard />', () => {
+  it('should not display if user has no copay statements or total is 0', () => {
+    const tree = renderWithStoreAndRouter(<CopaysCard />, {
+      initialState,
+    });
+
+    expect(tree.getByTestId('zero-debt-paragraph')).to.exist;
+    expect(
+      tree.getByText('Your total VA copay balance is $0', { exact: false }),
```

Comment:
yeah! i found this easier than fighting with this error:

> TestingLibraryElementError: Unable to find an element with the text: .... This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.